packages/core/fs/package.json

```diff
@@ -49,7 +49,6 @@
   },
   "dependencies": {
     "@parcel/rust": "2.12.0",
-    "@parcel/types": "2.12.0",
```

Comment:
Not sure this should have been removed as it's imported in `index.d.ts`

Reply:
You're right; I shouldn't have removed this.